modules/ROOT/pages/amazon/amazon-sqs-connector-xml-maven.adoc

```diff
@@ -0,0 +1,72 @@
+= Amazon SQS XML and Maven Support - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+If you manually code a Mule app in XML either from the Anypoint Studio XML editor or from a text editor, you can get access to Amazon SQS Connector in your app by adding reference statements to your XML Mule flow and in the Apache Maven `pom.xml` file.
```

Comment:
```suggestion
If you manually code a Mule app in XML either from the Anypoint Studio XML editor or from a text editor, you can get access to Anypoint Connector for Amazon SQS (Amazon SQS Connector) in your app by adding reference statements to your XML Mule flow and in the Apache Maven `pom.xml` file.
```

Reply:
Also, can we reword one or the other of "adding reference statements to your XML Mule flow" and the subtitle "Add a Namespace to the Connector"? Or is that connection obvious to your audience?